src/components/charts/entropy.js

```diff
@@ -8,30 +8,33 @@ import { changeColorBy } from "../../actions/colors";
 import { materialButton, materialButtonSelected } from "../../globalStyles";
 import EntropyChart from "./entropyD3";
 import InfoPanel from "./entropyInfoPanel";
-import { changeMutType } from "../../actions/treeProperties";
+import { TOGGLE_MUT_TYPE } from "../../actions/types";
+import { analyticsControlsEvent } from "../../util/googleAnalytics";
+import { modifyURLquery } from "../../util/urlHelpers";
 import "../../css/entropy.css";
 
-const calcEntropy = function (entropy) {
+const calcEntropy = function calcEntropy(entropy) {
```

Comment:
This can also be linted as:

```
const calcEntropy = (entropy) => {
```

which I think is cleaner.

Reply:
Although I see now that below you used

```
export const determineColorByGenotypeType = (colorBy) => {
```

So maybe you had a reason for the `const funcName = function (funcName) {` style?

Reply:
not here I didn't - it could be a fat arrow function.
There are cases, however, when you can't use it, namely if the function body uses `this` as the binding is different between `function () {}` and `() => {}`

Reply:
Cool. I didn't know about the `this` difference.